CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
   **FIXED**
   - Mastering workflow example was missing the generate clusters step, which has been rectified using proper endpoint
+  - `data` field renamed to `_data` (private). ([details](https://github.com/Datatamer/unify-client-python/issues/61))
```

Comment:
I see you copied the format some of the previous changelog entries.

I think we should move to a new format:
```suggestion
  - [#61](https://github.com/Datatamer/unify-client-python/issues/61) `data` field renamed to `_data` (private).
```

Reply:
~We should probably add this format in the changelog item within PR template~ specified the changelog entry format in #73